.github/workflows/BuildAndTestOnEveryPush.yml

```diff
@@ -33,12 +33,19 @@ jobs:
     - name: Build
       run: dotnet build
     
-    - name: Test
+    - name: Run ReadExchangeEmailTests
+      env:
+        Exchange_User_Password: ${{ secrets.EXCHANGE_USER_PASSWORD }}
```

Comment:
I think that this will cause passwords and such go into build logs. Check out this code line from Frends Tasks workflow that is masking sensitive information from logs: https://github.com/FrendsPlatform/FrendsTasks/blob/f1961cd090f9929eb675e02ada7a5c31198c52ea/.github/workflows/build_main.yml#L53

Reply:
They are not visible in the build logs: 
![image](https://user-images.githubusercontent.com/36008914/195017207-fe483863-6825-4c0e-ba70-79158194b2f3.png)